backend/index.js

```diff
@@ -21,7 +21,6 @@ app.get('/', (request, response) => {
 app.all('*', function (req, res, next) {
   let token = getBearerToken(req)
   if (token == SECRET) {
-    next() // Go onto the queries.
```

Comment:
This fixes an issue where the first request always fails. I added it to this pull request just to make it simpler to review.